website_sale_product_minimal_price/static/src/js/website_sale_product_minimal_price.js

```diff
@@ -0,0 +1,62 @@
+odoo.define("website_sale_product_minimal_price.shop_min_price", function(require) {
+    "use strict";
+
+    var publicWidget = require("web.public.widget");
+    var core = require("web.core");
+    const field_utils = require("web.field_utils");
+
+    publicWidget.registry.WebsiteSaleProductMinimalPrice = publicWidget.Widget.extend({
+        selector: "#products_grid",
+        xmlDependencies: [
+            "/website_sale_product_minimal_price/static/src/xml/website_sale_product_minimal_price.xml",
+        ],
+
+        start: function() {
+            return $.when.apply($, [
+                this._super.apply(this, arguments),
+                this.render_price(),
+            ]);
+        },
+        render_price: function() {
```

Comment:
To have a more readable source use 'await/async':
```suggestion
        render_price: async function() {
```